demos/travel-and-hospitality/alliance-ticket-sharing/uni_configuration/schema.json

```diff
@@ -0,0 +1,558 @@
+{
+  "$schema": "https://json-schema.org/draft-07/schema#",
+  "$id": "https://vendia.net/schemas/alliance-ticket-sharing/v1.0",
+  "title": "Alliance Partner Ticket Share",
+  "description": "A ticket data model with ACLs that allows alliance partners to share ticket data in real-time and with control",
+  "x-vendia-acls": {
+    "TicketAcl": {
+      "type": "Ticket"
+    }
+  },
+  "type": "object",
+  "properties": {
+    "Ticket": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "passengerNameRecordCode": {
+            "type": "string",
+            "description": "A unique passenger record, represented by a 6-digit alphanumeric value",
+            "pattern": "[A-Z0-9]{6}"
+          },
+          "airlineCode": {
+            "type": "string",
+            "description": "The 2-letter code of the issuing airline",
+            "pattern": "[A-Z]{2}"
```

Comment:
I know this is a demo with known inputs, but this character class will only allow 2 upper case letters. Two lower case letters will fail validation.